config/firewalld.service.in

```diff
@@ -17,6 +17,23 @@ StandardError=null
 Type=dbus
 BusName=org.fedoraproject.FirewallD1
 KillMode=mixed
+DevicePolicy=closed
+KeyringMode=private
+LockPersonality=yes
+MemoryDenyWriteExecute=yes
+NoNewPrivileges=yes
+PrivateDevices=yes
+ProtectClock=yes
+ProtectControlGroups=yes
+ProtectHome=yes
+ProtectHostname=yes
+ProtectKernelLogs=yes
+ProtectKernelModules=yes
+ProtectKernelTunables=yes
```

Comment:
firewalld does write to some `/proc/sys`, e.g. `/proc/sys/net/ipv4/ip_forward` to enable forwarding if masquerading is enabled.